### PR TITLE
first attempt at patching

### PR DESCRIPTION
**Pull request by binarytemple-external**

### CODE REVIEW

File: src/yz_extractor.erl [hunk 1]
```diff
@@ -31,6 +31,7 @@
 -define(DEFAULT_MAP, [{default, yz_noop_extractor},
                       {"application/json",yz_json_extractor},
                       {"application/riak_counter", yz_dt_extractor},
+                      {"application/riak_gset", yz_dt_extractor},
                       {"application/riak_map", yz_dt_extractor},
```
Comment (binarytemple-external) [inline src/yz_extractor.erl:35]: Wish this was more robust. the presumption that the list is sorted is a bit iffy.

Comment (zeeshanlakhani) [inline src/yz_extractor.erl:35]: agreed. I was bit by this too, haha.

Comment (binarytemple-external) [inline src/yz_extractor.erl:35]: 1 evening of my life gone, never to be regained. Thanks Ryan ;-)

Comment (lukebakken) [inline src/yz_extractor.erl:35]: Might as well test it!

basho/yokozuna#665

**Comment by bsparrow435** [comment]
@JeetKunDoug @fadushin thoughts on this PR for 2.2 inclusion?

**Comment by binarytemple-external** [comment]
@JeetKunDoug @fadushin @bsparrow435 - would save me resubmitting it for 2.3 inclusion, this P/R is pretty much entirely self contained.

**Comment by binarytemple-external** [comment]
Not any more, it's a world of conflicts. Will close and resubmit when the develop-2.3 opens up.

**Comment by JeetKunDoug** [comment]
@binarytemple-bet365 sorry about that - we were too close to release of 2.2 to try to get this incorporated/tested in time. Please do re-open when develop-2.3 happens, which shouldn't be much longer now.

**Pull request closed without merging**